receiver/oracledbreceiver/metadata.yaml

```diff
@@ -233,7 +233,6 @@ metrics:
     enabled: true
     gauge:
       value_type: int
-      input_type: string
```

Comment:
The logic now requires two DB values to be multiplied by each other. This means we have to manually convert both values to `int64`, multiply them, then use that result to set the value here. Converting back to string just to convert to `int64` again is unnecessary, so I've removed this option.